src/page/payment-step.page.ts

```diff
@@ -4,7 +4,7 @@ export class  PaymentStepPage {
   private payByBankOption: ElementFinder;
 
   constructor () {
-    this.payByBankOption = $('#HOOK_PAYMENT > div:nth-child(1) > div > p > a');
+    this.payByBankOption = $('.bankwire');
```

Comment:
did you validate this selector was unique?
To do validate: Search the selector using the brower console

Reply:
it seems unique. I would have used the title attribute, but this is fine, no need to change it